Makefile

```diff
@@ -129,6 +134,14 @@ run-local-db: check-env-run-local-db
 	  --env INFERENCE_MODEL=$(ANSIBLE_CHATBOT_INFERENCE_MODEL) \
 	  ansible-chatbot-stack:$(ANSIBLE_CHATBOT_VERSION)
 
+run-venv: check-env-run
+	LLAMA_STACK_PORT=$(LLAMA_STACK_PORT) \
+	VLLM_URL=$(ANSIBLE_CHATBOT_VLLM_URL) \
+	VLLM_API_TOKEN=$(ANSIBLE_CHATBOT_VLLM_API_TOKEN) \
+	INFERENCE_MODEL=$(ANSIBLE_CHATBOT_INFERENCE_MODEL) \
+	EMBEDDING_MODEL=./embeddings_model \
+	llama stack run ~/.llama/distributions/.venv/.venv-run.yaml
```

Comment:
our actual run is based on running our custom image, so this is changing a bit the picture? are you skipping the custom build this way?

Reply:
uv run --with llama-stack llama stack build

Reply:
> our actual run is based on running our custom image, so this is changing a bit the picture? are you skipping the custom build this way?

`run-venv` is for running `ansible-chatbot-stack` without using a container image for debugging purposes.  `llama stack build` supports three image types (`conda`, `container`, `venv`) and it is for the `venv` image type.